spec/controllers/shared_examples/shared_spec_for_sync_controller.rb

```diff
@@ -453,6 +453,25 @@ def discard_patient(record)
           expect(response_record_ids).to match_array block_records.map(&:id)
           expect(non_block_records).not_to include(*response_record_ids)
         end
+
+        it "sync facility group records when user is a teleconsult MO" do
+          allow_any_instance_of(User).to receive(:can_teleconsult?).and_return true
+          block_records = Timecop.travel(15.minutes.ago) {
+            create_record_list(2, patient: patient_in_same_block, facility: facility_in_same_block)
+          }
+          non_block_records = Timecop.travel(15.minutes.ago) { create_record_list(2, facility: facility_in_other_block) }
```

Comment:
The last record in the first request is repeated as the first record in the second request. This happens because of the `>=` comparison [here](spec/controllers/shared_examples/shared_spec_for_sync_controller.rb).

Reply:
Ooh fun. Could we make that clearer in the comment?